component-samples/demo/README.MD

```diff
@@ -286,6 +286,12 @@ curl -D - --digest -u apiUser:${api_password} --location --request POST 'http://
 --header 'Content-Type: text/plain' --data-binary '@<path-to-file>'
 ```
 
+**NOTE:** If a failure is observed in the middle of SVI file transmission, there is a possibility that runnning container might hit memory limit set in the respective docker-compose.yml file.
```

Comment:
Do we want to say in case of large payload size?

Reply:
Updated.